docs/user/guides/build/index.rst

```diff
@@ -0,0 +1,34 @@
+How-to guides: build process
+============================
+
+⏩️ :doc:`Setup build notifications and webhooks </guides/build-notifications>`
+    Build notifications can alert you when your builds fail so you can take immediate action.
+    In this guide,
+    you will learn how to get notified through various available channel integrations,
+    including email and chat.
```

Comment:
I feel like we could also just pull this content from the top of the guide, instead of writing it a bunch of different places?

```suggestion
⏩️ :doc:`/guides/build-notifications`
   .. include:: /guides/build-notifications.rst
      :start-line: 3
      :number-lines: 3
```

https://docutils.sourceforge.io/docs/ref/rst/directives.html#toc-entry-37

Reply:
If this makes sense to you, I can futz with it to get it working. I think we could even parse out the `How to` probably from the heading with some magic, which seems worth doing if we're having to repeat ourselves 75 times.

Reply:
Oh nice idea! Do you think we can use this "at scale"?

Reply:
It'd be very interesting to know if this can work. If it's just 5% better than repeating the same text, I don't think we need it... but if there's something here that can also be used in the Feature Reference, then I think we're on to something.

I haven't been able to find a good solution, so I started dreaming up a Sphinx extension that can handle meta data similar to how Hugo does [Front Matter](https://gohugo.io/content-management/front-matter/).

Reply:
Sphinx should be able to support metadata by default. I think we can definitely come up with a better approach here.

Reply:
If you can show me how you'd like it to work, I'll see if I can make it happen tomorrow.

Reply:
I haven't found a way to insert the metadata? Do you know?

```
:short_howto_title: This is a shorter title
:howto_intro: Here we show you a way to achieve something
```

Reply:
It needs to be possible to insert the metadata into the contents of the page itself but also in other pages.

Reply:
Other pages is going to be hard. How does Hugo handle referencing other pages metadata?

Reply:
The short answer is shortcodes:

https://stackoverflow.com/a/62425260

It's pretty close to what we would want a sphinx-extension providing a role or directive for doing.

Reply:
We should definitely be able to make Sphinx do this without a ton of work. I really dislike all this repeating and substringing of content. It's much better for our tooling to do this for us, instead of making each of us do a bunch of extra work.

Reply:
Yes, haha, totally agree, I just feel like all paths took me towards "write your own extension" :)

Reply:
Hoping that we will learn more about potential solutions in [the new issue](https://github.com/readthedocs/readthedocs.org/issues/10157) :)